rfq/cli.go

```diff
@@ -0,0 +1,61 @@
+package rfq
+
+import (
+	"fmt"
+)
+
+const (
+	MockPriceOracleServiceAddress = "use_mock_price_oracle_service_" +
+		"promise_to_not_use_on_mainnet"
+)
+
+// CliConfig is a struct that holds tapd cli configuration options for the RFQ
+// service.
+//
+// nolint: lll
+type CliConfig struct {
+	PriceOracleAddress string `long:"priceoracleaddress" description:"Price oracle gRPC server address (rfqrpc://<hostname>:<port>). Mock address: use_mock_price_oracle_service_promise_to_not_use_on_mainnet"`
```

Comment:
Suggestion for description suffix: "To use the integrated mock, use the following value: "?